docs/getting-started/first-steps.md

```diff
@@ -11,7 +11,7 @@ alias:
     Make sure you have read [[getting-started#installation]] before using the library. 
     In particular read about which extra dependencies you may need.
 
-## Main concepts
+## Main Concepts
```

Comment:
Why this change? We don't capitalize titles elsewhere, do we?

Reply:
We actually do in quite a lot of places. The inconsistency highly depends on who wrote the documentation. I will make everything consistent.